tests/test/cache/basic.tests.js

```diff
@@ -1,18 +1,19 @@
 /*
```

Comment:
I have to say I am not a fan of
`*.test.js` = Javascript file with tests
`*.tests.js` = Javascript file with test utilities

would something like `basic.test.helper.js` be a clearer division?